### DISCUSSION TITLE
I need help idk how to fix my problem and was just messing around with the settings.

### DISCUSSION THREAD

**Discussion by Soapium** [pear-devs/pear-desktop#2719] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/0d904b21-ffb7-4cff-ba7a-2a21010b2f42)

So Idk what happened to the plugins, or the options on the top I don't quite remember what I did but I was messing with the settings on the top or the plugins can't remember, I tried reinstalling it but I think it saves my settings even after uninstalling, I think I'm using the portable one idk I'm using the youtube-music-3.6.2-x64.nsis.7z 
I want the plugin list and everything else to appear but Idk how to bring it back. has anyone done something like I've done and was able to fix it? again I tried reinstalling it, checking if there was a setting to change it back to default settings, and I think I was also seeing if there were any control + buttons but I've pressed a lot and tried pressing the F1-etc buttons to see if the plugin and the other settings to appear at the top but nothing has appeared.

**Comment by Lockszmith-GH** [comment, marked as answer]
Does [this](https://github.com/th-ch/youtube-music?tab=readme-ov-file#why-apps-menu-isnt-showing-up) help?

**Reply by Soapium** [reply]
I forgot what menu button I pressed so I didn't know if I used hide menu but after pressing left alt everything appeared, thanks for the help I feel kinda dumb finding out it was on the bottom, thanks for the help, you've resolved my issue.

**Reply by MissKalunji** [reply]
Thank you! panicked wondering on to enable it lol
